### ISSUE TITLE
Try Dropout

### ISSUE THREAD

**Issue by felixwege** [felixwege/Schlaeferprojekt#5]
"Dropout is a regularization technique for reducing overfitting in neural networks by preventing complex co-adaptations on training data." [1]

[1] https://en.wikipedia.org/wiki/Dropout_(neural_networks)

**Comment by felixwege** [comment]
This is only applicable once we have our first NN.

**Comment by felixwege** [comment]
This is only applicable one we have our first NN.
